src/core/src/main/resources/org/apache/jmeter/resources/messages_de.properties

```diff
@@ -224,6 +224,11 @@ graph_results_ms=Millisekunden (ms)
 graph_results_no_samples=Anzahl der Proben
 graph_results_throughput=Durchsatz
 graph_results_title=Ergebnisse
+graphql_http_sampler_title=GraphQL HTTP Request
```

Comment:
You don't need to provide those messages_XX.properties if you don't translate them.
Only messages.properties and messages_fr.properties are mandatory

Reply:
Fixed: fcf3ed2fc (Just removed the unnecessary, untranslated ones).
Thanks!